pallet-chainlink-feed/src/feed.rs

```diff
@@ -0,0 +1,195 @@
+//! utils for creating feeds
+
+use crate::{BalanceOf, Config, FeedConfig, RoundId};
+use frame_support::{
+	sp_runtime::traits::{One, Zero},
+	Parameter,
+};
+use sp_std::prelude::*;
+
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+
+/// Builder with all the parameters to call the `create_feed`
```

Comment:
Any hints on how to test this @mattsse?

Reply:
@ebarakos This allows creating feeds that should be included in the genesis block with the feed pallet's genesisconfig:

https://github.com/smartcontractkit/chainlink-polkadot/pull/79/files#diff-c5eb2db45eadd23bc84e52e146f5af5c01907ae3ccd2b563f928ee67b565490aR169-R173

This builder helps creating those like in this test

https://github.com/smartcontractkit/chainlink-polkadot/pull/79/files#diff-34ba86f45aaa350811973171ddae78d326f79cb5b184943e213971dd291f20b0R1252-R1272

This allows creating feeds directly in the chainspec:
https://github.com/smartcontractkit/chainlink-polkadot/blob/master/substrate-node-example/node/src/chain_spec.rs#L163-L165